lib/slimmer.rb

```diff
@@ -47,4 +47,5 @@ module Processors
 
   class TemplateNotFoundException < StandardError; end
   class CouldNotRetrieveTemplate < StandardError; end
+  class IntermittentRetrievalError < StandardError; end
```

Comment:
Minor thought, I wonder if this should be a subclass of CouldNotRetrieveTemplate - feels like a specialisation of it rather than a distinct option.

Reply:
Oh yes, that makes sense. I've pushed up another commit for that.